drivers/hmis/app/models/hmis/hud/custom_service.rb

```diff
@@ -10,6 +10,7 @@ class Hmis::Hud::CustomService < Hmis::Hud::Base
   include ::HmisStructure::Service
   include ::Hmis::Hud::Concerns::Shared
   include ::Hmis::Hud::Concerns::EnrollmentRelated
+  include ::Hmis::Hud::Concerns::ClientProjectEnrollmentRelated
```

Comment:
This is great. All of the below models are enrollment-related and should include this concern so they have the same behavior:
```
Hmis::Hud::Disability,
Hmis::Hud::EmploymentEducation,
Hmis::Hud::Exit,
Hmis::Hud::Funder,
Hmis::Hud::HealthAndDv,
Hmis::Hud::IncomeBenefit,
Hmis::Hud::Service,
Hmis::Hud::CustomAssessment,
Hmis::Hud::CurrentLivingSituation,
Hmis::Hud::YouthEducationStatus,
```

Reply:
I added the concern to these models except for the Hmis::Hud::Funder, that caused test failures. Many of these models did not already have a project association or method.

Reply:
Whoops, Funder shouldn't have it. My bad. The rest should. Thanks!